src/components/layouts/ScrollToTop.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+
+import ArrowUp from '../../assets/icons/icons-arrow-up.png';
+
+const ScrollToTop = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  const toggleVisibility = () => {
+    if (window.scrollY > 50) {
```

Comment:
Can we check against a more responsive unit of measuring instead of pixels? If not that's ok. Whenever possible try to use responsive units.